application/apps/indexer/parsers/src/dlt/fmt.rs

```diff
@@ -559,6 +588,36 @@ impl<'a> fmt::Display for FormattableMessage<'a> {
                     None => write!(f, "[Unknown CtrlCommand]"),
                 }
             }
+            PayloadContent::NetworkTrace(slices) => {
+                self.write_app_id_context_id_and_message_type(f)?;
+
+                if let Some(ext_header) = &self.message.extended_header {
+                    let msg_type = &ext_header.message_type;
+                    if MessageType::NetworkTrace(NetworkTraceType::Ipc) == *msg_type
+                        || MessageType::NetworkTrace(NetworkTraceType::Someip) == *msg_type
+                    {
+                        if let PayloadContent::NetworkTrace(slices) = &self.message.payload {
+                            if let Some(slice) = slices.get(1) {
+                                if let Ok((_, message)) = SomeipParser::parse_message(
+                                    self.fibex_someip_metadata,
+                                    slice,
+                                    None,
+                                ) {
+                                    return write!(
+                                        f,
+                                        "{}",
+                                        format!("{message}").replace(SOMEIP_COLUMN_SENTINAL, " ")
+                                    );
+                                }
+                            }
+                        }
+                    }
+                }
```

Comment:
I tried to reduce the nesting here and came up with: 
```rust 
let is_someip_trace =
    self.message
        .extended_header
        .as_ref()
        .is_some_and(|ext_header| {
            matches!(
                ext_header.message_type,
                MessageType::NetworkTrace(NetworkTraceType::Ipc)
                    | MessageType::NetworkTrace(NetworkTraceType::Someip)
            )
        });

if is_someip_trace {
    if let PayloadContent::NetworkTrace(slices) = &self.message.payload {
        if let Some(Ok((_, message))) = slices.get(1).map(|slice| {
            SomeipParser::parse_message(self.fibex_someip_metadata, slice, None)
        }) {
            return write!(
                f,
                "{}",
                format!("{message}").replace(SOMEIP_COLUMN_SENTINAL, " ")
            );
        }
    }
}
```
You can apply part of it, if you think would have improvements

Reply:
Used parts of it, but also now checking on errors in SomeipParser::parse_message now.